README.md

```diff
@@ -1,2 +1,141 @@
-# sagastandard
-One Standard to bring them all, and in the darkness bind them.
+# Javascript Saga-Standard style
+### One Standard to bring them all, and in the darkness bind them.
+
+Based on [feross/standard] and [Flet/semistandard] with a few additionnal opiniated style rules.
+
+## Install
+```bash
+npm install -g sagastandard
+```
+
+## Rules
+Importantly:
+- **1tbs**: check [one-true-brace-style](http://eslint.org/docs/rules/brace-style)
+- **curly**: always require braces
+- **max-len at 80**: except for urls.
+- **NO console.log**
+- **NO debugger;***
+- **semicolons** check [Flet/semistandard]
+- Check [feross/standard] for the rest of the rules.
+
+## Usage
+
+The easiest way to use JavaScript Saga-Standard Style to check your code is to install it globally as a Node command line program. To do so, simply run the following command in your terminal (flag `-g` installs `sagastandard` globally on your system, omit it if you want to install in the current working directory):
+
+```bash
+npm install sagastandard -g
+```
+
+After you've done that you should be able to use the `sagastandard` program. The simplest use case would be checking the style of all JavaScript files in the current working directory:
+
+```
+$ sagastandard
+Error: Use JavaScript Saga-Standard Style
+  lib/torrent.js:950:11: Expected '===' and instead saw '=='.
+```
+
+### Editor plugins
+
+There are currently no editor plugins directly supporting Saga-Sandard, but the following plugins can be customized to use it.
+
+- **Sublime users**: Try [SublimeLinter-contrib-semistandard](https://github.com/Flet/SublimeLinter-contrib-semistandard) for linting in your editor!
+- **Atom users** - Install [linter-js-standard](https://atom.io/packages/linter-js-standard)
+
```

Comment:
can't we add something like : `ln -s /usr/local/bin/sagastandard /usr/local/bin/semistandard` to this section of the documentation? I know for sure that Sublime plugin is based on CLI calls. I don't know for Atom